try:
    [1, 2][2]
except IndexError:
    print("IndexError n√∫mero fuera de rango")
else:
    print("Exception Otro error diferente")

